voter/src/main/java/org/terracotta/voter/ActiveVoter.java

```diff
@@ -209,6 +215,22 @@ public void registerAndHeartbeat(ExecutorService executorService, ClientVoterMan
     LOGGER.warn("Rejected by all servers. Attempting to re-register");
   }
   
+  private static void sleepFor10() {
+    try {
+      TimeUnit.SECONDS.sleep(10);
+    } catch (InterruptedException ie) {
+      //
```

Comment:
Not sure if this should be ignored especially since we need to support interruption in `deregister`.

Reply:
will reset interrupt.